backend/app/user/routes.py

```
@@ -55,3 +66,62 @@
+    background_tasks.add_task(
+        user_service.update_user_avatar_url, user_id, urls.get("full")
+    )
+
+    return ImageUploadResponse(
+        success=True, urls=urls, message="Avatar uploaded successfully."
+    )
+
+
+@router.delete("/me/avatar", response_model=DeleteUserResponse)
+async def delete_user_avatar(
+    current_user: dict = Depends(get_current_user),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)